apis/metabase.graphql

```diff
@@ -1074,19 +935,23 @@ type OpenIdConnectToken {
 
 type OpticalData implements Data {
   appliedMethod: AppliedMethod!
-  componentId: Uuid!
+  component: Component
+  componentId: UUID!
   description: String
+  infraredEmittances: [Float!]!
   name: String
+  nearnormalHemisphericalSolarReflectances: [Float!]!
+  nearnormalHemisphericalSolarTransmittances: [Float!]!
```

Comment:
Thanks to the update, `make example` validates `queries/metabase/tutorial.graphql` succesfully. Relates to #262